src/hooks/useWindowSpecs.ts

```diff
@@ -0,0 +1,21 @@
+import { useEffect, useState } from 'react'
+
+const useWindowSpecs = (): { innerWidth: number; innerHeight: number } => {
+  const [innerWindowSpecs, setInnerWindowSpecs] = useState({
+    innerWidth: window.innerWidth,
+    innerHeight: window.innerHeight,
```

Comment:
Accessing, `window.innerWidth` on each render causes a [reflow](https://gist.github.com/paulirish/5d52fb081b3570c81e3a#window), which is unnecessary evil.
Do like [this](https://usehooks.com/useWindowSize/)